resources/testharnessreport.js

```diff
@@ -28,6 +28,15 @@ function dump_test_results(tests, status) {
                 stack: status.stack};
     results_element.textContent = JSON.stringify(data);
 
+    if (tests.length === 1) {
```

Comment:
Is this entirely for debugging and intended to be reverted, or is it an important part of this patch?

Reply:
I did it to ensure this was working.
I do not strongly care about this particular change but think that this might be convenient for people running tests in a browser.
If we go with implicit test(), this change is meaningless.
Otherwise, providing additional information to people running tests might be useful.